sdk/test/test_e2e.py

```diff
@@ -546,3 +549,23 @@ async def scrape(sdk: SDK, url, context) -> None:
     )
     assert observer.data[0]["status"] == "Open"
     assert len(observer.data[0]["attachments"]) == 4
+
+
+async def test_403_status_on_goto(server, observer):
```

Comment:
@pytest.mark.parametrize("harness", [playwright_harness, soup_harness])

make sure this works in soup too

Reply:
i didn't want to make it work initially since the types can be a bit sus but updated now